PyInstaller/utils/win32/winutils.py

```diff
@@ -65,13 +56,14 @@ def extend_system_path(paths):
 
     Some hooks might extend PATH where PyInstaller should look for dlls.
     """
+    from ... import compat
     old_PATH = compat.getenv('PATH', '')
     paths.append(old_PATH)
     new_PATH = os.pathsep.join(paths)
     compat.setenv('PATH', new_PATH)
 
 
-def import_pywin32_module(module_name):
+def import_pywin32_module(module_name, _is_venv=None):
```

Comment:
Adding an internal-only parameter is necessary to prevent circular imports, since this function is called from `compat`